One test failing with 4.0.0: `Arrays are not almost equal to 5 decimals`

While updating the package to 4.0.0 in Fedora rawhide (the dev branch), we see one test failing:


```
>       assert_almost_equal(s.volume, 3160.274957542371, decimal=5)
tests/core/test_soma.py:263: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
args = (3160.2749000561025, 3160.274957542371), kwds = {'decimal': 5}
    @wraps(func)
    def inner(*args, **kwds):
        with self._recreate_cm():
>           return func(*args, **kwds)
E           AssertionError: 
E           Arrays are not almost equal to 5 decimals
E            ACTUAL: 3160.2749000561025
E            DESIRED: 3160.274957542371
/usr/lib64/python3.12/contextlib.py:81: AssertionError
```

The values are close, but not as close as they should be. How should we handle this please? Complete log is here:

[fedora-rawhide-neuroml-build-log.txt](https://github.com/BlueBrain/NeuroM/files/15483281/fedora-rawhide-neuroml-build-log.txt)

--- a/tests/core/test_soma.py
+++ b/tests/core/test_soma.py
@@ -223,7 +223,7 @@ def test_Soma_Cylinders():
     # area of a cylinder (excluding end caps) is:
     # 2*pi*r*h == 4*pi*r^2 == area of a sphere of radius 20
     assert s.radius == 20.0
-    assert_almost_equal(s.area, 5026.548245743669)
+    assert_almost_equal(s.area, 5026.548245743669, decimal=5)
     assert_array_equal(s.center, [0, 0, -10])
     assert 'SomaCylinders' in str(s)
 
@@ -241,7 +241,7 @@ def test_Soma_Cylinders():
 
     assert 'SomaNeuromorphoThreePointCylinders' in str(s)
     assert list(s.center) == [0.0, 0.0, 0.0]
-    assert_almost_equal(s.area, 1256.6370614)
+    assert_almost_equal(s.area, 1256.6370614, decimal=5)
 
     # some neuromorpho files don't follow the convention
     # but have (ys + rs) as point 2, and have xs different in each line
@@ -260,7 +260,7 @@ def test_Soma_Cylinders():
     assert 'SomaNeuromorphoThreePointCylinders' in str(s)
     assert list(s.center) == [0.0, 0.0, 0.0]
     assert_almost_equal(s.area, 794.76706126368811, decimal=5)
-    assert_almost_equal(s.volume, 3160.274957542371, decimal=5)
+    assert_almost_equal(s.volume, 3160.274957542371, decimal=3)
 
     s = load_morphology(
         StringIO(
@@ -276,7 +276,7 @@ def test_Soma_Cylinders():
     ).soma
 
     assert list(s.center) == [0.0, 0.0, 0.0]
-    assert_almost_equal(s.area, 444.288293851)  # cone area, not including bottom
+    assert_almost_equal(s.area, 444.288293851, decimal=5)  # cone area, not including bottom
 
 
 def test_soma_overlaps():

